packages/flame/lib/src/components/custom_painter_component.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/widgets.dart';
+
+import '../extensions/vector2.dart';
+import 'position_component.dart';
+
+/// A [PositionComponent] that renders a [CustomPainter] at the designated
+/// position, scaled to have the designated size and rotated to the specified
+/// angle.
+///
+/// This component is usefull to [CustomPainter]s between your Flutter UI and Flame game.
+class CustomPainterComponent extends PositionComponent {
```

Comment:
We should highlight that should repaint is ignored.

Reply:
Done